bigtable/api/SnippetsTest/runTest.ps1

```diff
@@ -11,4 +11,6 @@
 # WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 # License for the specific language governing permissions and limitations under
 # the License.
-dotnet test --test-adapter-path:. --logger:junit -v n
\ No newline at end of file
+
+    dotnet restore --force
```

Comment:
Is the extra spacing here deliberate, or just copied from the runTests script?

Reply:
That's just a copy paste thing, fixing ...